cmds/rm/rm.go

```diff
@@ -12,6 +12,7 @@
 //     -v: verbose mode
 //     -R: remove file hierarchies
 //     -r: equivalent to -R
+//     -f: ignore nonexistent files and never prompt
```

Comment:
You appear to have overlooked the synopsis above: ` //     rm [-Rrvi] FILE...`

Reply:
Ignoring os.IsPermission(err) is expected behavior of `rm -f` based on other `rm` implementations. This PR looks good for the functionality added. We can either add ignoring os.IsPermission(err) in a separate PR or here. Let me know which makes more sense.

Reply:
Oops. I've changed it in the `cmd` declaration and completely missed this one. Will fix it, thanks.